x/evm/genesis_test.go

```diff
@@ -96,6 +96,24 @@ func (suite *EvmTestSuite) TestInitGenesis() {
 			},
 			true,
 		},
+		{
+			"ignore empty account code checking",
+			func() {
+				acc := suite.app.AccountKeeper.NewAccountWithAddress(suite.ctx, address.Bytes())
+
+				suite.app.AccountKeeper.SetAccount(suite.ctx, acc)
```

Comment:
Set a non-empty code hash to the account?

Reply:
Yes, the codehash in the account keeper will be an empty code hash in this test case. However, when we set the Account code to empty in the evm state. No matter whether the codehash is empty or not, it always passes the check.

Reply:
Because empty code hash also pass the test even without this change, right? So a non-empty one can demonstrate the change works?

Reply:
I will try to add another one with non-empty hash case